src/content/products/deepsparse/enterprise.mdx

```diff
@@ -300,7 +300,7 @@ Use Case: A workload that might benefit from the elastic scheduler is one in whi
 
 ### License
 
-The Community Edition of the project's binary containing the DeepSparse Engine is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
+The Community Version of the project's binary containing DeepSparse is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
```

Comment:
https://github.com/neuralmagic/docs/pull/117 has the updated verbiage.

Reply:
When we merge the branches, we will take 117's verbiage, dont need to do it here